spec/models/claims/state_machine_spec.rb

```diff
@@ -297,24 +297,24 @@
 
     context 'update last_submitted_at on redetermination or await_written_reasons' do
       it 'set the last_submitted_at to the current time for redetermination' do
-        current_time = Time.now
+        current_time = Time.now.change(usec: 0)
         claim.submit!
         claim.allocate!
         claim.refuse!
 
-        Timecop.freeze 6.months.from_now do
+        travel_to current_time + 6.months do
```

Comment:
[ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3627/files#r538245188)